### PR TITLE
Fixed formatting for urlsession.yml

### PR DESCRIPTION
**Pull request by jawa9000**
Fixed a few formatting issues with the ti.urlsession that was slipping through our yaml linting process.

### CODE REVIEW

File: apidoc/urlsession.yml [hunk 1]
```diff
@@ -2,78 +2,79 @@
 name: Modules.URLSession
 summary: Wrapper to support iOS's NSURLSession class for background downloads.
 description: |
-    These APIs are supported on iOS 7 and later.
+  These APIs are supported on iOS 7 and later.
```
Comment (hansemannn) [inline apidoc/urlsession.yml]: This is incorrect. It has to be 4 spaces like all other docs. If the doc build succeeds with this setting, it has an issue in the generator.

**Comment by hansemannn** [comment]
@jawa9000 What to do with this PR? I still think we need 4 spaces after breaking the line, e.g.
```yml
  - name: Test
     description: |
         Hello world
```
instead of
```yml
  - name: Test
     description: |
       Hello world
```

**Comment by jawa9000** [comment]
Ping!

**Comment by hansemannn** [comment]
@jawa9000 For what? We cannot merge the changes so far, they have incorrect indentation (2 vs. 4 like all our other docs).

**Pull request closed without merging**